src/testhost.x86/DefaultEngineInvoker.cs

```diff
@@ -228,7 +230,8 @@ private static void InitializeEqtTrace(IDictionary<string, string> argsDictionar
             // Setup logging if enabled
             if (argsDictionary.TryGetValue(LogFileArgument, out string logFile))
             {
-                EqtTrace.InitializeVerboseTrace(logFile);
+                var traceLevelInt = CommandLineArgumentsHelper.GetIntArgFromDict(argsDictionary, TraceLevelArgument);
```

Comment:
verify for invalid tracelevel  values

Reply:
Added check for invalid tracelevel values 
Verified with older testhost as well.